test/tests/WriteTest.cpp

```diff
@@ -176,4 +177,17 @@ struct WriteTest : tpunit::TestFixture {
         tester->executeWaitVerifyContent(query2);
     }
 
+    void keywordsAsValue() {
+        SData query("query: INSERT INTO stuff VALUES ( NULL, 11, 'Please update the test' );");
+        tester->executeWaitVerifyContent(query);
+
+        SData query2("query: INSERT INTO stuff VALUES ( NULL, 12, 'Do not delete the test' );");
+        tester->executeWaitVerifyContent(query2);
+
+        // Monil B - 05/26/21
```

Comment:
no need for this comment (git blame does this for us 🙃 )

Reply:
Okay removing this.